helm/charts/determined/templates/genai/genai-deployment.yaml

```diff
@@ -21,6 +21,14 @@ spec:
     spec:
       priorityClassName: determined-system-priority
       serviceAccount: genai-{{ .Release.Name }}
+      {{- if .Values.genai.agentGroupID }}
+      {{ $gid := .Values.genai.agentGroupID }}
+      securityContext:
+        runAsUser: {{ $gid }}
```

Comment:
user isn't always the same id as group id. this might have to be a different parameter